### ISSUE TITLE
4g-coverage

### ISSUE THREAD

**Issue by RedWalters** [GSS-Cogs/provider-ons-ess#43]

**Comment by RedWalters** [comment]
I can change the Config to remove the Mission and Category columns if we want that

**Comment by RedWalters** [comment]
Draft [Here](https://beta.gss-data.org.uk/admin/drafts/6f68fd00-348a-4074-9907-9658ab00a878/claim)

**Comment by canwaf** [comment]
Yup. The metadata standards are going to be changing from the ESS side since they've now learnt of the possibilities of the Description field being Markdown.

Regardless of the metadata improvements, the mission and category dimensions should be excluded from the CSV-W.
